neqo-transport/src/cc/classic_cc.rs

```diff
@@ -613,7 +613,6 @@ mod tests {
     };
 
     const PTO: Duration = RTT;
-    const RTT_ESTIMATE: RttEstimate = RttEstimate::from_duration(RTT);
```

Comment:
`RttEstimate::from_duration` is no longer `const` as it uses `Default::default` of `PerrAckDelay`.

This affects test code only, thus simply replacing this constant with `RttEstimate::default`.

Reply:
Can we make it `#[cfg(test)]` in that case?